src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -10,9 +10,9 @@
   --pf-c-modal-box--MaxHeight: calc(100% - var(--pf-global--spacer--2xl)); // MaxHeight ensures that the modal will not go off the screen and instead the body will scroll
   --pf-c-modal-box--m-align-top--spacer: var(--pf-global--spacer--sm);
   --pf-c-modal-box--m-align-top--xl--spacer: var(--pf-global--spacer--xl);
-  --pf-c-modal-box--m-align-top--MarginTop: var(--pf-c-modal-box--m-align-top--spacer);
```

Comment:
should you remove this variable at a breaking change ?

Reply:
Oh good catch, changed it back and left a note to rename it.